frontend/javascripts/oxalis/model/sagas/load_histogram_data_saga.ts

```diff
@@ -1,88 +1,99 @@
-import type { HistogramDataForAllLayers } from "oxalis/store";
+import type { HistogramDataForAllLayers, DatasetLayerConfiguration } from "oxalis/store";
 import type { Saga } from "oxalis/model/sagas/effect-generators";
 import { select } from "oxalis/model/sagas/effect-generators";
-import { call, take, put } from "typed-redux-saga";
+import { call, take, takeEvery, put } from "typed-redux-saga";
 import {
   setHistogramDataAction,
+  setHistogramDataForLayerAction,
   updateLayerSettingAction,
+  type ReloadHistogramAction,
 } from "oxalis/model/actions/settings_actions";
 import { getHistogramForLayer } from "admin/admin_rest_api";
 import DataLayer from "oxalis/model/data_layer";
 import Model from "oxalis/model";
-import type { APIDataset } from "types/api_flow_types";
+import type { APIHistogramData } from "types/api_flow_types";
 
-async function fetchAllHistogramsForLayers(
-  dataLayers: Array<DataLayer>,
-  dataset: APIDataset,
-): Promise<HistogramDataForAllLayers> {
-  const histograms: HistogramDataForAllLayers = {};
+export default function* loadHistogramDataSaga(): Saga<void> {
+  yield* take("WK_READY");
+  yield* takeEvery("RELOAD_HISTOGRAM", reloadHistogramForLayer);
+  const dataLayers: Array<DataLayer> = yield* call([Model, Model.getColorLayers]) as any;
```

Comment:
```suggestion
  const dataLayers: Array<DataLayer> = yield* call([Model, Model.getColorLayers]);
```

I think, it is not necessary anymore because of TS (at least I just tested this on another branch).